spec/swagger_schemas/models/participants/enrolment.rb

```diff
@@ -0,0 +1,164 @@
+      type: :boolean,
+      example: true
+    },
+    sparsity_uplift: {
+      description: "Indicates whether this participant qualifies for an uplift payment due to sparsity",
+      type: :boolean,
+      example: true
+    },
+    schedule_identifier: {
+      description: "The schedule of the participant",
+      type: :string,
+      enum: Schedule.identifiers.keys,
+      example: "ecf-standard-january"
+    },
+    delivery_partner_id: {
+      description: "Unique ID of the delivery partner associated with the participant",
+      type: :string,
+      format: :uuid,
+      example: "42a9ef2f-9059-400a-92ff-4830a629d0c5"
+    },
+    withdrawal: {
+      description: "The details of a participant withdrawal",
+      type: :object,
+      required: %I[reason date],
+      properties: {
+        reason: {
+          description: "The reason a participant was withdrawn",
+          type: :string,
+          enum: TrainingPeriod.withdrawal_reasons.keys.map(&:dasherize),
+          example: "moved-school"
+        },
+        date: {
+          description: "The date and time the participant was withdrawn",
+          type: :string,
+          format: "date-time",
+          example: "2021-05-31T02:22:32.000Z"
+        }
+      }
+    },
+    deferral: {
+      description: "The details of a participant deferral",
+      type: :object,
+      required: %I[reason date],
+      properties: {
+        reason: {
+          description: "The reason a participant was deferred",
+          type: :string,
+          enum: TrainingPeriod.deferral_reasons.keys.map(&:dasherize),
+          example: "career-break"
+        },
+        date: {
+          description: "The date and time the participant was deferred",
+          type: :string,
+          format: "date-time",
+          example: "2021-05-31T02:22:32.000Z"
+        }
+      }
+    },
+    created_at: {
+      description: "The date and time the participant was created",
+      type: :string,
+      format: "date-time",
+      example: "2023-01-01T00:00:00Z"
+    },
+    induction_end_date: {
+      description: "The induction period end date",
+      type: :string,
+      format: "date",
+      example: "2023-01-01",
+      nullable: true
+    },
+    mentor_funding_end_date: {
+      description: "The participant mentor training completion date",
+      type: :string,
+      format: "date",
+      example: "2023-01-01",
+      nullable: true
+    },
+    cohort_changed_after_payments_frozen: {
+      description: "Identify participants that migrated to a new cohort as payments were frozen on their original cohort",
+      type: :boolean,
+      example: true
+    },
+    mentor_ineligible_for_funding_reason: {
+      description: "The reason why funding for a mentor's training has ended",
+      type: :string,
+      enum: Teacher.mentor_became_ineligible_for_funding_reasons.keys + %w[mentor_not_completed],
```

Comment:
Are we supporting the `mentor_not_completed` reason?

I can't find any references to it in this codebase 🤔

Reply:
the only ones we support are:
```
  - completed_declaration_received
  - completed_during_early_roll_out
  - started_not_completed
```
not sure where `mentor_not_completed` came from?